azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -83,6 +86,9 @@ protected void configure() {
     bind(DataSource.class).to(AzkabanDataSource.class);
     bind(ExecutorManager.class).in(Scopes.SINGLETON);
     bind(AlerterHolder.class).in(Scopes.SINGLETON);
+    bind(CommonMetrics.class).in(Scopes.SINGLETON);
```

Comment:
Since CommonMetrics is annotated as singleton already, I think you don't need this line here. same with MetricsManager.